website/assets/css/page.css

```diff
@@ -10,6 +10,34 @@ h1, h2, h3, h4, h5, h6 {
   font-size: .9rem;
 }
 
+.navbar {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding: 0rem 0rem;
+    height:74px;
+}
+
+.bg-dark {
+    background-color: #000!important;
+}
+
+p {
+    margin-top: 0px;
+    margin-bottom: rem;
+}
+
+.jumbotron {
+  background: #135CC0;
+  color: #FFFFFF;
+  max-width: 100%;
+  height:220px;
+  padding-bottom: 0px
+  margin-bottom: 1rem;
+  border-radius: 0!important;
+}
+
+
```

Comment:
may want to clean-up extra whitespace here

Reply:
No, It doesn't need.